### PR TITLE
Add feature page for support for protocol for Ingress, Egress Gateway

### PR DESCRIPTION
**Pull request by johnma14**

### CODE REVIEW

**Comment by brian-avery** [comment]
@istio/wg-networking-maintainers can you please review?

**Comment by johnma14** [comment]
@howardjohn @istio/wg-networking-maintainers can we get a review on this one. Thank you.

File: features/ingress-egress-gateway.md [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+
+[//]: # (The feature implementation section must be completed before submission of the document.)
+
+# Feature:
+
+[//]: # (All information in this section is mandatory.)
+
+**Feature name:**
+
+Gateways: Ingress, Egress for all protocols
```
Comment (howardjohn) [inline features/ingress-egress-gateway.md:17]: "all protocols" doesn't really make sense here. We can't possibly ever support "all". I would just make it "gateways"

File: features/ingress-egress-gateway.md [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+
+**Short description:**
+
+Support Ingress and Egress traffic for all protocols
+
+[//]: # (A short description of the feature. One or two sentences maximum.)
+
+
+**Design Docs:**
+[BTS for gateway](https://docs.google.com/document/d/1DxwqhNqxm1r3fSomyQgJWy103bv29B0ZTEvu_hZqQgc/edit?resourcekey=0-uPN2Gtx-BHvj-sAvN9SGKA#)
```
Comment (howardjohn) [inline features/ingress-egress-gateway.md:33]: It feels odd to have an un-implemented future looking doc as the core design doc for a feature implemented + stable over 3 years ago.

Comment (johnma14) [reply features/ingress-egress-gateway.md]: Removed it. Thanks John

File: features/ingress-egress-gateway.md (hunk 1 above)
Comment (johnma14) [reply features/ingress-egress-gateway.md]: Updated the feature name.

**Pull request closed without merging**